lib/tasks/evm_dba.rake

```diff
@@ -195,9 +179,10 @@ namespace :evm do
 
         puts "Initializing region and database..."
         # Create the region from our REGION file, initialize a new miq_database row for this region
-        EvmDba.run_cmd_with_open4("bin/rails runner 'MiqDatabase.seed; MiqRegion.seed;'")
+        AwesomeSpawn.run!("bin/rails runner", :params => ["MiqDatabase.seed; MiqRegion.seed"])
       rescue => err
-        STDERR.puts "Encountered issue setting up Database using region #{region}: #{err}\n"
+        message = err.kind_of?(AwesomeSpawn::CommandResultError) ? err.error : err.message
```

Comment:
Keep in mind, the most likely place you'd see this is when you region the database from the appliance_console which requires that you're at the machine or ssh'd into it, I think it's safe for any errors to be printed.  In general, we'd have to think how we use AwesomeSpawn to figure out what types of things would be bad to print.